modules/proc-switching-bf2-nic.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * networking/switching-bf2-nic-dpu.adoc
+
+:_content-type: PROCEDURE
+[id="proc-switching-bf2-nic_{context}"]
+= Switching Bluefield-2 from DPU mode to NIC mode
+
+Use the following procedure to switch Bluefield-2 from data processing units (DPU) mode to network interface controller (NIC) mode.  
+
+[IMPORTANT]
+====
+Currently, only switching Bluefield-2 from DPU to NIC mode is supported. Switching from NIC mode to DPU mode is unsupported. 
+====
+
+.Prerequisites
+
+* You have installed the SR-IOV Operator. For more information, see "Installing SR-IOV Network Operator". 
```

Comment:
"SR-IOV Operator" to "SR-IOV Network Operator"

Reply:
Good pedantic catch